docs/home/installing-tracking-script.md

```diff
@@ -0,0 +1,17 @@
+## How to install the tracking script
+
+After installation and setup of the cron jobs you're ready to begin tracking contacts. You need to add a simple piece of JavaScript to the websites for each site you wish
+to track via Mautic. This is a very simple process and you can add this tracking script to your website template file,
```

Comment:
We don't use terms like 'simple' - we use the Google Technical Writing Style Guide which explains more: https://developers.google.com/style/tone?hl=en

```suggestion
to track via Mautic. This is straightforward process, and you can add this tracking script to your website template file,
```